xm-commons-exceptions/src/main/java/com/icthh/xm/commons/exceptions/BusinessNotFoundException.java

```diff
@@ -0,0 +1,27 @@
+package com.icthh.xm.commons.exceptions;
+
+import java.util.Map;
+
+/**
+ * Extended exception from {@link BusinessException} with response code 404
+ * @author maximbogun
+ */
+public class BusinessNotFoundException extends BusinessException {
```

Comment:
I would recommend renaming to smth like NotFoundBusinessException.

Reply:
Our proposal to stay on BusinessNotFoundException,- means that those exception subtype of BusinessException